templates/zerver/api/check-narrow-matches.md

```diff
@@ -0,0 +1,35 @@
+# Check narrow matching messages
+
+{generate_api_description(/messages/matches_narrow:get)}
+
+## Usage examples
+
+{start_tabs}
+{tab|python}
+
+{generate_code_example(python)|/messages/matches_narrow:get|example}
+
+{tab|curl}
+
+{generate_code_example(curl)|/messages/matches_narrow:get|example}
+
+{end_tabs}
+
+## Arguments
+
+{generate_api_arguments_table|zulip.yaml|/messages/matches_narrow:get}
+
+## Response
+
+#### Return values
+
+* `messages`: A dictionary of messages matching the narrow with message ids
+    as keys and `message` objects as values with the given fields:
+    * `match_content`: Contents of the message that matches the narrow.
+    * `match_topic`: Topic of the message matched.
```

Comment:
Can you move these data to the openapi file?  And rebase this onto master.

Reply:
I don't think you understood I meant by move here -- since @orientor added support for this a few weeks ago, we don't want any data on return values to be here in these files.  We should just include the rendered data from the OpenAPI file here using our markdown extension.

Sorry, I'd thought I'd mentioned you on the PR that made this change and migrated the content.

Reply:
@timabbott Actually, I tried checking that by removing this from the .md file and I think it didn't render so I placed it back.
Anyway, I'll remove it.👍